controllers/ingressnodefirewallconfig_controller.go

```diff
@@ -154,9 +154,6 @@ func (r *IngressNodeFirewallConfigReconciler) syncIngressNodeFwConfigResources(c
 			if len(config.Spec.NodeSelector) > 0 {
 				ds.Spec.Template.Spec.NodeSelector = config.Spec.NodeSelector
 			}
-			if len(config.Spec.Tolerations) > 0 {
-				ds.Spec.Template.Spec.Tolerations = config.Spec.Tolerations
```

Comment:
hmm I thought that was the default ? will set whatever default we need in the ds template instead of the controller, will check and update the PR accordingly

Reply:
Sure the template works as well. I'm pretty sure that the default is no toleration for pods, so any taint will affect them; should be easy to verify